web/beacon-app/src/features/projects/components/ProjectsTable.tsx

```diff
@@ -11,10 +12,27 @@ type ProjectTableProps = {
 };
 
 const initialColumns = [
-  { Header: 'Project ID', accessor: 'id' },
-  { Header: 'Project Name', accessor: 'name' },
+  { Header: t`Project Name`, accessor: 'name' },
   {
-    Header: 'Date Created',
+    Header: t`Description`,
+    accessor: (p: Project) => {
+      const description = p?.description;
+      if (!description) {
+        return '---';
+      }
+      // cut off description at 100 characters
+      return description?.length > 100 ? `${description?.slice(0, 100)}...` : description || '---';
+    },
+  },
+  {
+    Header: 'Status',
```

Comment:
The Figma design has icons next to each status. Ex. A green circle icon if the status is Active. Will that be added in a separate story?